src/runtime/cmath.py

```diff
@@ -250,3 +250,51 @@ def tanh(x: c64) -> c64:
 @overload
 def tanh(x: c32) -> c32:
     return _lfortran_ctanh(x)
+
+@ccall
+def _lfortran_zpolar(x: c64) -> c64:
+    pass
+
+@ccall
+def _lfortran_cpolar(x: c32) -> c32:
+    pass
+
+@overload
+def polar(x: c64) -> c64:
+    return _lfortran_zpolar(x)
+
+@overload
+def polar(x: c32) -> c32:
+    return _lfortran_cpolar(x)
```

Comment:
These won't work as we don't have these functions in `lfortran_intrinsic.h`.

Reply:
Where can I find the functions included in lfortran_intrinsic.h?

Reply:
In `src/libasr/runtime/lfortran_intrinsic.h/cpp`.